jobs/inprogress_update/inprogress_update.py

```diff
@@ -31,39 +37,65 @@ def get_ops_params():
-
     return client_delay, examine_delay, max_rows
 
 
+def publish_email_message(payload: dict):
+    """Publish the email message onto the NATS emailer subject."""
+    subject = current_app.config.get('NATS_EMAILER_SUBJECT', 'entity.email')
+    queue.publish_json_to_subject_sync(payload, subject)
+    current_app.logger.debug('publish to queue successfully, subject: %s, event:%s', subject, payload)
+
+
+def furnish_request_message(
+        request: Request,
+        option: str
+):  # pylint: disable=redefined-outer-name
+    """Send notification info to the mail queue."""
+    current_app.logger.debug('Start of the furnishing of request for %s nrNum=%s', option, request.nrNum)
+    payload = create_cloud_event_msg(
+        msg_id=str(uuid.uuid4()),
+        msg_type='bc.registry.names.request',
+        source=f'/requests/{request.nrNum}',
+        time=datetime.utcfromtimestamp(time.time()).replace(tzinfo=timezone.utc).isoformat(),
+        identifier=request.nrNum,
+        json_data_body={
+            'request': {
+                'nrNum': request.nrNum,
+                'option': option
+            }
+        }
+    )
+    current_app.logger.debug('About to publish email for %s nrNum=%s', option, request.nrNum)
+    publish_email_message(payload)
+
+
 def inprogress_update(user: User, max_rows: int, client_delay: int, examine_delay: int) -> (int, bool):
     """Update inprogress nrs."""
     row_count = 0
 
     try:
         # for nrs edited by clients
+        # pylint: disable=C0121
         client_edit_reqs = db.session.query(Request). \
             filter(Request.stateCd == State.INPROGRESS). \
-            filter(Request.lastUpdate <= text(f'(now() at time zone \'utc\') - INTERVAL \'{client_delay} SECONDS\'')). \
+            filter(Request.lastUpdate <= text(f"(now() at time zone 'utc') - INTERVAL \'{client_delay} SECONDS\'")). \
             filter(Request.checkedOutBy != None). \
             order_by(Request.lastUpdate.asc()). \
             limit(max_rows). \
             with_for_update().all()
-        for r in client_edit_reqs:
+        for request in client_edit_reqs:
             row_count += 1
 
-            current_app.logger.debug(f'processing: {r.nrNum}')
-            current_app.logger.debug(f'nr {r.nrNum}, state: {r.stateCd} last_update:{r.lastUpdate}')
+            current_app.logger.debug(f'processing: {request.nrNum}')
+            current_app.logger.debug(f'nr {request.nrNum}, state: {request.stateCd} last_update:{request.lastUpdate}')
 
-            r.stateCd = State.DRAFT
-            r.checkedOutBy = None
-            db.session.add(r)
-            errors = nro.checkin_checkout_nr(r, 'UNLOCK')
-            if errors:
-                raise RuntimeError('Failed to update nro.')
-            # commit here to keep this entry in sync with NRO (in case errors happen later)
-            db.session.commit()
+            furnish_request_message(request, 'clients-edit-inprogress')
```

Comment:
same comment as below regarding line 144(`furnish_request_message(request, 'nro-updating')`).  this won't be picked up by emailer properly.

Reply:
as per our conversation, emailer corresponding updates will be made as part of another ticket.